docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1327,6 +1327,311 @@ The ACS will be used for V&V testing. It is defined as:
 - Includes mail/online, telephone, and personal visits 
 
 
+Taxes
+^^^^^
+
+Taxes, as we all know, can contain many different forms and processes. 
+For this model, we will split the tax information into two main sections: 
+W2/1099 forms from employers; and 1040 forms from simulants. We will look 
+at these separately, starting with W2 and 1099 forms. 
+
+
+W2 and 1099 Forms
+'''''''''''''''''
+
+**When to Sample** 
+
+- Sample compiled on the time step containing Jan 1st of each year (the time step might end on Jan 2nd, Jan 15th, Jan 27th, etc.)
+- However, we will want to track every job a simulant has had for any time step within a calendar year, which might require additional observers. If a simulant changes jobs in March of 2020, their tax documents on Jan 1st of 2021 will need to include both their current job, and their job from February of 2020. Jobs can be tracked for complete time steps, so through the last day of the time step containing Jan 1st. 
+
+**What to Sample** 
+
+.. list-table:: Simulant Attribute to Sample 
+  :widths: 20
+  :header-rows: 0
+
+  * - Unique simulant ID (for PRL tracking)
+  * - First name
+  * - Middle initial 
+  * - Last name
+  * - Age 
+  * - DOB 
+  * - Mailing Address 
+  * - Social Security Number 
+  * - Wages (income from this job)
+  * - Employer ID 
+  * - Employer Name 
+  * - Employer Address 
+  * - Employer Zip Code 
+  * - Type of Tax Form (W2 or 1099)
+
+If a simulant does not have a social security number but is 
+employed, they will need this number to be filled in. If there 
+is a person in their household who has a SSN, use this number 
+instead. If there is not a person in their household with a SSN 
+then fill in a random number. This is designed to reflect 
+undocumented immigrants who might use fake or no longer 
+valid SSNs to obtain employment. 
+
+For this observer, a new row should be made for each **employment**, not 
+each simulant. This means that a simulant can have multiple rows of 
+data, or just one row of data. 
+
+Here is an example: 
+
+.. image:: W2_example.PNG
+
+**Who to Sample** 
+
+Everyone who has had an employer listed within the current calendar year 
+will receive either a W2 or a 1099 form. For those with multiple jobs during 
+the year, they will be duplicated and receive multiple forms. We currently 
+will not model persistence from year to year on which type of form. 
+
+
+The rate of the the types of forms are below. This data is 
+from a review of 2016 tax data by [Lim_2019]_ . 
+
+.. list-table:: Percent W2 versus 1099  
```

Comment:
i think this is probably per job, but just to make sure: is this percent per job, or percent per person?

i.e., if someone had two jobs, do we select if they file a w2 vs 1099 for their first job with 94.65 v. 5.35 % probability, and then independently sample again for their second job? or do we sample once, and then all of one persons jobs return the same type of form?

Reply:
Added, thanks! We can select per job.